uikit/common/sdk/src/main/java/com/sygic/ui/common/sdk/utils/DefaultValues.kt

```diff
@@ -0,0 +1,3 @@
+package com.sygic.ui.common.sdk.utils
+
+const val POSITION_ON_MAP_ENABLED_BASE_DEFAULT_VALUE = false
```

Comment:
uff, toto je dost haluzna konstanta. potrebujeme ju realne? aj tak to vzdy bude false a neviem ci to niekedy chceme menit

Reply:
je, jen jsem chtěl aby to bylo konzistentní a až to bude někdo někdy měnit, tak aby se to nastavilo na více místech jednou konstantou. Nikdy neříkej nikdy, víš jako to bylo s Hvožďarou a s vyžadováním permission hned při startu app...

Reply:
chapem, len je to take divne, ze vsetky default ma ten component a tento zrazu niekde v common :/ Navyse myslim, ze tu fakt to prepojenie je trosku zbytocne a ked budu oba false tak to bude ok

Reply:
oki, odstraním